main.go

```diff
@@ -94,44 +88,43 @@ func handler(w http.ResponseWriter, r *http.Request, logger log.Logger) {
 		authName = "public_v2"
 	}
 
-	moduleName := query.Get("module")
-	if len(query["module"]) > 1 {
-		http.Error(w, "'module' parameter must only be specified once", http.StatusBadRequest)
-		snmpRequestErrors.Inc()
-		return
-	}
-	if moduleName == "" {
-		moduleName = "if_mib"
+	queryModule := query["module"]
+	if len(queryModule) == 0 {
+		queryModule = append(queryModule, "if_mib")
 	}
-
 	sc.RLock()
 	auth, authOk := sc.C.Auths[authName]
-	module, moduleOk := sc.C.Modules[moduleName]
-	sc.RUnlock()
 	if !authOk {
+		sc.RUnlock()
 		http.Error(w, fmt.Sprintf("Unknown auth '%s'", authName), http.StatusBadRequest)
 		snmpRequestErrors.Inc()
 		return
 	}
-	if !moduleOk {
-		http.Error(w, fmt.Sprintf("Unknown module '%s'", moduleName), http.StatusBadRequest)
-		snmpRequestErrors.Inc()
-		return
+	uniqueM := make(map[string]bool)
+	var modules []*collector.NamedModule
+	for _, qm := range queryModule {
+		for _, m := range strings.Split(qm, ",") {
+			module, moduleOk := sc.C.Modules[m]
```

Comment:
Perhaps skip if m=="", in order to handle cases like `foo,,bar` ?  And then at the very end, `if len(modules)==0 { modules=[]string{"if_mib"} }` (instead of lines 92-93)